.github/workflows/integrationtests.yaml

```diff
@@ -84,17 +62,18 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+
+      - uses: actions/setup-java@v2
         with:
+          distribution: 'temurin'
           java-version: '11'
+          cache: 'gradle'
 
-      - name: AWS S3 StatusChecker Test
+      - name: AWS S3 Tests
         env:
-          RUN_INTEGRATION_TEST: true
           S3_ACCESS_KEY_ID: root
           S3_SECRET_ACCESS_KEY: password
-        run: ./gradlew extensions:aws:s3:s3-provision:check
+        run: ./gradlew -p extensions/aws/s3 test -DIncludeTags="AwsS3IntegrationTest"
 
       - name: AWS S3 StatusChecker Test
         env:
```

Comment:
at line 80 there's a `RUN_INTEGRATION_TEST` variable definition, that's not needed anymore

Reply:
Updated. Thanks. Missed due to merge.